.jenkins/pytorch/common_utils.sh

```diff
@@ -49,6 +49,12 @@ function assert_git_not_dirty() {
     fi
 }
 
+function pip_install_whl() {
+  # This is used to install PyTorch and other build artifacts wheel locally
+  # without using any network connection
+  pip install --no-index --no-deps "$@"
```

Comment:
Nit (or use pip3, so that it would work on machines that has python2 and python3 installed at the same time)
```suggestion
  python3 -mpip install --no-index --no-deps "$@"
```